apps/chat-e2e/src/testData/conversationHistory/conversationData.ts

```diff
@@ -215,17 +215,18 @@ export class ConversationData extends FolderData {
       conversation,
       replayIndex,
     );
+    const conversationCopy = JSON.parse(
```

Comment:
Can we somehow omit the creation of the deep copy here? Why do we need it here?

Reply:
`conversation.messages` and `conversation.replay.replayUserMessagesStack` objects cannot refer to the same object that's why a copy is required